gateway/src/App.js

```diff
@@ -7,149 +7,120 @@ import "react-bootstrap-typeahead/css/Typeahead.bs5.css";
 import "react-bootstrap-typeahead/css/Typeahead.css";
 
 import { sanitizeUrl } from "@braintree/sanitize-url";
-import {
-  LivepeerConfig,
-  createReactClient,
-  studioProvider,
-} from "@livepeer/react";
 import { useAccount, useInitNear } from "near-social-vm";
 import {
-  Link,
-  Route,
-  BrowserRouter as Router,
-  useLocation,
+	Link,
```

Comment:
Can you add a prettier to this repo? Let's use this one: https://github.com/PotLock/bos-app/blob/main/.prettierrc